R/collect.R

```diff
@@ -55,6 +55,24 @@
 #' [collect_extracts()] returns a tibble with columns for the resampling
```

Comment:
I think this part could put an emphasis on "gives you access to the fitted workflows" by, e.g., starting the content description with that.

Reply:
maybe move this into the section below (which could resolve the comment I made about the section title)?

Reply:
Attempted to resolve in [21932aa](https://github.com/tidymodels/tune/pull/743/commits/21932aaa897dc4941f660d3f1a41d219c874caef). I opted not to move this into the section below since that section is reused in the control docs, but will hear you out if you think that's how we ought to do it!